Views/MainView.py

```diff
@@ -0,0 +1,94 @@
+from tkinter import Tk, Canvas, Label, N, S, E, W, Button
+
+class MainView(Tk):
+    class Constants:
+        title = "Pizarra Màgica"
+        heigth = 600
+        width = 500
+        screen_heigth = 600
+        screen_width = 1000
+        button_heigth = 150
+        button_width = 100
+        center = N + S + E + W
+        right = W
+        bar_offset = 30
+        font_family = 'Arial'
+        font_size = 30
+        value_x = 0
+        value_y = 500
+        first_y = 600
+        color = 'Black'
+        event= "<Button-1>"
+        event_delete = "<space>"
+
+        @classmethod
+        def size(cls):
+            return "{}x{}".format(cls.screen_width, cls.screen_heigth)
+
+
+
+    def __init__(self):
+        super().__init__()
+        self.title(self.Constants.title)
+        self.geometry(self.Constants.size())
+
+        self.__lines = True
+        self.__canvas = Canvas(self, width = self.Constants.width , height=self.Constants.heigth)
+        self.__green = Button(self, text = "Green",bg = '#00ff00', font =self.Constants.font_family  )
+        self.__blue = Button(self, text="Blue", bg = '#0000ff', font=self.Constants.font_family)
```

Comment:
Me estas repitiendo mucho codigo solo para hacer botones, no seria mas facil un for?